hive-metastore/src/main/java/org/apache/iceberg/hive/HiveOperationsBase.java

```diff
@@ -62,6 +82,48 @@ interface HiveOperationsBase {
 
   String table();
 
+  String catalogName();
+
+  ContentType contentType();
+
+  default Table loadHmsTable() throws TException, InterruptedException {
+    try {
+      return metaClients().run(client -> client.getTable(database(), table()));
+    } catch (NoSuchObjectException nte) {
+      LOG.trace(
+          "{} not found {}", contentType(), catalogName() + "." + database() + "." + table(), nte);
```

Comment:
```suggestion
          "{} not found {}", contentType(), database() + "." + table(), nte);
```

this is just TRACE logging, so it should be ok to not have the catalog. I'm not even sure that logging stmt should even exist